client/components/App.jsx

```diff
@@ -0,0 +1,30 @@
+    // Stretch goal: refactor into additional sub-components/rethink file system
+    // with potential nesting for sub-categories of components
+
+import React from 'react';
+import Header from './Header';
+import Filters from './Filters';
+import ReviewList from './ReviewList';
+import Footer from './Footer';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { var: 3};
```

Comment:
This is the restaurant ID that will be passed into App's props in the final version, right?